packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -85,6 +87,13 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    * @default {false}
    */
   @api multilineFooter;
+  /**
+   * Whether the generated answer should be collapsible when it exceeds the maximum height of 250px.
+   * @api
+   * @type {boolean}
+   * @default {false}
+   */
+  @api collapsible = false;
```

Comment:
Just a minor suggestion for the name, `isCollapsible` sounds slightly better for me as a boolean proeprty..

Reply:
Even when it's going to be used in the HTML?

`<c-quantic-generated-answer is-collapsible` ?